.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -1,17 +1,15 @@
-Link to the milestone on Github e.g. https://github.com/nimblehq/git-templates/milestone/41?closed=1
-or
-Link to the project management tool for the release
+https://github.com/nimblehq/flutter-templates/milestone/?
 
 ## Features
 
 Provide the ID and title of the issue in the section for each type (feature, chore and bug). The link is optional.
```

Comment:
Shall we remove this line too?

Reply:
@lydiasama We will need to update the git template later when separating the `root template` and `base template project`. No worries 🙏